.drone.star

```diff
@@ -23,16 +28,6 @@ config = {
                 "7.4",
             ],
         },
-        "reducedDatabases": {
-            "phpVersions": [
-                "7.3",
-            ],
-            "databases": [
-                "mysql:8.0",
-            ],
-            "coverage": False,
-        },
-    },
```

Comment:
It looks like the script deleted too many lines.

Reply:
fixed